.codebuild/scripts/lint_pr.sh

```diff
@@ -1,6 +1,8 @@
 set -xeo pipefail
 # extract the PR number from the PR link
 PR_NUM=${CODEBUILD_WEBHOOK_TRIGGER##*/}
+PROJECT_USERNAME=aws-amplify
+REPO_NAME=amplify-codegen
```

Comment:
These two can be removed as they are already included in the CodeBuild build environment vars. (can be found in CDK code)

Reply:
Was it a recent CDK update? I ran this one today morning and it failed to resolve these variables correctly.

Reply:
Looks like these params are defined for PR project but for release. I will take my words back. Either approach works for me.

Reply:
This fails on the deployment project which doesn't have these as ENV vars.